parts/k8s/kubernetesagentresourcesvmas.t

```diff
@@ -1,5 +1,5 @@
     {
-{{if .IsAcceleratedNetworkingEnabled}}
+{{if .AcceleratedNetworkingEnabled}}
```

Comment:
See comment below, the function was superfluous, we can just check the bool type instead